### PR TITLE
Add TG4X rev 2.1 shield

### PR DESCRIPTION
**Pull request by mubeenkhan94**
Thanks to the ZMK Discord for the help with this! This works using Bluetooth and has a Colemak Mod Dhm keymap by default.

### CODE REVIEW

File: app/boards/shields/tg4x/tg4x.keymap [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    };
+};
+
+/ {
+    keymap {
+        compatible = "zmk,keymap";
+
+        default_layer {
+            bindings = <
+    &kp GRAV   &kp Q       &kp W      &kp F         &kp P           &kp B    &kp J    &kp L      &kp U      &kp Y      &kp SCLN    &kp BKSP
```
Comment (petejohanson) [inline app/boards/shields/tg4x/tg4x.keymap:34]: Interesting keymap!

We try to have ZMK come with "stock" keymaps that are more basic/standardized, and then let user's override them with user configs to tweak to their liking.

Can we get a stock QWERTY version for here, and then I'm happy on Discord to help walk you through building locally with an "out of tree" keymap in a user config, so you can have the keymap you want.

**Review by petejohanson — CHANGES_REQUESTED** [review]
Just one comment on the "stock" keymap in the PR.

Thanks for creating this!

File: app/boards/shields/tg4x/tg4x.keymap (hunk 1 above)
Comment (mubeenkhan94) [reply app/boards/shields/tg4x/tg4x.keymap:34]: Sorry I was AWOL due to work. I will get this put together and submit this week.

**Comment by mubeenkhan94** [comment]
The keymap has been revised to QWERTY :)

File: app/boards/shields/tg4x/tg4x.keymap (hunk 1 above)
Comment (mubeenkhan94) [reply app/boards/shields/tg4x/tg4x.keymap]: Revised keymap

**Review by petejohanson — APPROVED** [review]
Thanks!

**Pull request merged**